README.md

```diff
@@ -1,6 +1,11 @@
 Python WVA Library
 ==================
 
+[![Build Status](https://travis-ci.org/digidotcom/python-wvalib.svg?branch=master)](https://travis-ci.org/digidotcom/python-wvalib)
+[![Latest Version](https://pypip.in/version/wva/badge.svg)](https://pypi.python.org/pypi/wva/)
+[![Supported Python versions](https://pypip.in/py_versions/wva/badge.svg)](https://pypi.python.org/pypi/wva/)
+[![License](https://pypip.in/license/devicecloud/badge.svg)](https://pypi.python.org/pypi/devicecloud/)
```

Comment:
Ah, good catch.  That's where I copied from but I forgot to replace one reference.  I'll fix that.